.env.example

```diff
@@ -16,3 +16,4 @@ SERVICE_PUSHER_BEAMS_INSTANCE_ID=
 SERVICE_PUSHER_BEAMS_SECRET_KEY=
 SNAPSHOT_URI=http://localhost:3000
 DEFAULT_NETWORK=1
+SNAPSHOT_WEBHOOK_URL=https://snapshot-discord.herokuapp.com
```

Comment:
This should be empty by default, and if not exist we shouldnt send any event, (for testnet for example we dont need that)